dietpi/preboot

```diff
@@ -299,12 +128,6 @@
 
 		fi
```

Comment:
All the above as well should go into the new script, shouldn't it?

Reply:
I tough those had to be run at every boot ?

Reply:
@FredericGuilbault 
Nope, `if (( $G_DIETPI_INSTALL_STAGE == -1 )); then` means fresh image, the firstrun step contains `echo 0 /DietPi/dietpi/.install_stage` which is read into `G_DIETPI_INSTALL_STAGE`.